sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -23,6 +23,8 @@ initscript {
     def pluginRepositoryUrl = getInputParam('gradle.plugin-repository.url')
     def gePluginVersion = getInputParam('develocity.plugin.version')
     def ccudPluginVersion = getInputParam('develocity.ccud-plugin.version')
+    def gradlePluginRepoUsername = getInputParam('gradle.plugin-repository.username')
+    def gradlePluginRepoPassword = getInputParam('gradle.plugin-repository.password')
```

Comment:
I would rename the variables as `pluginRepositoryUsername` and `pluginRepositoryPassword` to be consistent with the existing `pluginRepositoryUrl`. 

Then I would reorder the variable definitions so all three are next to each other.

Reply:
Done